### DISCUSSION TITLE
no output for LSF jobs during training

### DISCUSSION THREAD

**Discussion by minwang-ai** [Lightning-AI/pytorch-lightning#10321] [DDP / multi-GPU / multi-node]
Hi,

I submit a IBM LSF jobs to GPU cluster but after auto learning rate search, there is no output.
Please find the attached zip file for output.
Could you please tell me why?

Best wishes,
MIn
[211102_221500_YCB_train_SS_baseline_2381987.err.zip](https://github.com/PyTorchLightning/pytorch-lightning/files/7464285/211102_221500_YCB_train_SS_baseline_2381987.err.zip)

**Comment by akihironitta** [comment]
@MinWang1997 Could you paste the output here in text instead of in zip file?
